commands/project/upload.ts

```diff
@@ -128,6 +131,11 @@ exports.builder = yargs => {
       type: 'string',
       default: '',
     },
+    translate: {
+      hidden: true,
+      type: 'boolean',
+      default: false,
+    },
```

Comment:
I added this to `hs project upload` but not `hs project watch` primarily because `hs project watch` gets almost no usage, and there were some other road bumps, such as `hs project watch` doesn't work unless your project already has a successful build, which is not currently achievable with the new system.

I can add the options to `hs project watch` if we think it is worthwhile, but the command won't work until build scheduling is in place and component teams have begun to implement their builds.